src/co/elastic/NotificationManager.groovy

```diff
@@ -74,18 +85,26 @@ def analyzeFlakey(Map params = [:]) {
     def numberOfCreatedtedIssues = 0
     tests.each { k, v ->
       def issue = v
-      if (!v?.trim()) {
-        def issueDescription = buildTemplate([
+      def issueDescription = buildTemplate([
           "template": 'flaky-github-issue.template',
           "testName": k,
           "jobUrl": boURL,
-          "testData": testsErrors?.find { it.name.equals(k) }
-        ])
+          "PR": env.CHANGE_ID?.trim() ? "#${env.CHANGE_ID}" : '',
+          "commit": env.GIT_BASE_COMMIT?.trim() ?: '',
+          "testData": testsErrors?.find { it.name.equals(k) }])
+      if (v?.trim()) {
+        try {
+          issueWithoutUrl = v.startsWith('https') ? v.replaceAll('.*/', '') : v
+          githubCommentIssue(id: issueWithoutUrl, comment: issueDescription)
+        } catch(err) {
+          log(level: 'WARN', text: "Something bad happened when commenting the issue '${v}'. See: ${err.toString()}")
+        }
+      } else {
         def title = "Flaky Test [${k}]"
         try {
           if (numberOfCreatedtedIssues < numberOfSupportedIssues) {
             retryWithSleep(retries: 2, seconds: 5, backoff: true) {
-              issue = githubCreateIssue(title: title, description: issueDescription, labels: labels, returnStdout: true)
+              issue = githubCreateIssue(title: title, description: issueDescription, labels: labels)
```

Comment:
`returnStdout` is not required anymore since it has been migrated to `gh` instead `hub`